tests/pytest_framework/src/common/blocking.py

```diff
@@ -23,21 +23,19 @@
 
 import time
 
-MONITORING_TIME = 10  # 10 sec
-POLL_FREQ = 0.001  # 1 millisecond
+DEFAULT_TIMEOUT = 10  # 10 sec
+DEFAULT_POLL_FREQ = 0.001  # 1 millisecond
```

Comment:
Do we plan to change `DEFAULT_POLL_FREQ` anywhere? If not I believe the `DEFAULT_` part is unnecessary. Although, I can live with this. :)

Reply:
Yes, I believe we will. We already do this in the unit tests, but in the future I think the timeout parameter will be exposed up until the user API. For example, for now read_logs waits for 10s unconditionally, but later this will be probably changed. Depending on a scenario, 10s maybe too small, or unnecessarily high, and we need to let the user specify the timeout.

So I would leave the name as it is.

Reply:
Whops I misread the request. POLL_FREQ is not changed indeed, nor think it will. I rename it back.
It would be the best eliminate altogether, and use some kind of exponential backoff algorithm instead. Polling continuously for 10 seconds with milliseconds frequency seems impractical.